services/minespace-web/src/tests/components/Forms/projects/projectSummary/DocumentUpload.spec.js

```diff
@@ -24,7 +26,12 @@ beforeEach(() => {
 
 describe("DocumentUpload", () => {
   it("renders properly", () => {
-    const component = shallow(<DocumentUpload {...dispatchProps} {...props} />);
+    const component = shallow(
```

Comment:
Tara's correct that this might open up a whole can of worms, but it might be worth trying render from react-testing library.

The syntax is something like this:

```const { container } = render(...)

expect(container.firstChild).toMatchSnapshot();```

The can of worms part comes from the fact that it will try to fully render the component including child components, but is more reliable to use as well.